roles/agent/tasks/agent_present.yml

```diff
@@ -1,9 +1,12 @@
 ---
 # present tasks file for agent
-- name: agent_present | Assert agent_public_key is set
+- name: agent_present | Assert authentication method is provided
   ansible.builtin.assert:
-    that: agent_public_key != ""
-    fail_msg: Provide a public key to be used by the Beszel binary agent.
+    that: ((agent_public_key != "") and (agent_token != "")) or (agent_universal_token is defined and agent_universal_token != "")
+    fail_msg: |
+      Provide either:
+      - Both agent_public_key AND agent_token (for individual system authentication), or
+      - agent_universal_token (for universal token authentication, Beszel v0.12.0+)
```

Comment:
Please could you add to this check that a user doesn't try to specify both the `agent_token` and `agent_universal_token` at the same time?

I know it's unlikely as the documentation is extensive but this is to ensure we cover all bases and make sure the user doesn't provide both which right now would get templated in the systemd unit file with both.